### PR TITLE
add middrange detector

### PR DESCRIPTION
**Pull request by jmborr**
**Description of work**
Include the MIDRANGE detector to BIOSANS instrument IDF.
References [EWM Task 1550 Add midrange panel to biosans_geometry.py](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=1550)

**Mid-Range Detector Info**

- Composed of eight 8-packs arrayed in an arc of circunference. Each 8-pack made of a front and a back 4-pack, intercalated.

- The first tube is located most East (positive X-axis) and is in the front panel. The first tube in the back panel is also located most East among all the tubes of the back panel. Detector indexes start at the first tube and then continue along the tubes in the front panel. Then continue at the first tube of the back panel and progresses along the tubes of the back panel. This sequence is analog to that of the WING and MAIN panels. Thus, the last tube in the detector is the last tube in the back panel, which is the most Westward tube.

- A distance of 4000 mm between the center of the sample and the center of "tube 16 of the front panel"  the arc passing in between the front and back panels of the detector

- An entry in the metadata will specify the angular displacement of the detector from its default angular position. In the default angular position, the beam axis passes through the middle of the last tube.

**To test:**
Just make sure you can `LoadEmptyInstrument` file BIOSANS_Definition.xml

(There is no associated issue.)

---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

**Review by backmari — DISMISSED** [review]
Tested in Mantid Workbench that I am able to run LoadEmptyInstrument successfully with the changed file BIOSANS_definition.xml.

**Review by backmari — DISMISSED** [review]
I am able to load all 4 changed XML files using LoadEmptyInstrument

**Review by gemmaguest — DISMISSED** [review]

File: docs/source/release/v6.7.0/SANS/New_features/35520.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+- Midrange detector included in BIOSANS instrument definition file.
```
[the remark this answers is not in the document]
Comment (jmborr) [reply docs/source/release/v6.7.0/SANS/New_features/35520.rst]: @gemmaguest  thanks for the tip. I just removed the release notes.
So, if I understand you correctly, the initializing step that downloads instruments will take care of fetching even the IDF's that were produced after the release of the particular version I'm running. Is this correct?

**Review by backmari — DISMISSED** [review]

File: docs/source/release/v6.7.0/SANS/New_features/35520.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterfpeterson) [reply docs/source/release/v6.7.0/SANS/New_features/35520.rst]: Exactly. However, I still prefer including the release notes so people stand a better chance of finding the instrument. ORNL puts the geometry in the nexus file which means the instrument download mechanism is not used.

**Review by backmari — APPROVED** [review]
Side-by-side view tested OK

**Pull request merged**